library/ssl_misc.h

```diff
@@ -1567,6 +1567,13 @@ static inline void mbedtls_ssl_handshake_set_state( mbedtls_ssl_context *ssl,
     ssl->state = ( int ) state;
 }
 
+/*
+ * Fetch TLS 1.3 handshake message header
+ */
+int mbedtls_ssl_tls13_fetch_handshake_msg( mbedtls_ssl_context *ssl,
+                                       unsigned hs_type,
+                                       unsigned char **buf,
+                                       size_t *buf_len );
```

Comment:
`fetch_handshake_msg` has been added at [here](https://github.com/ARMmbed/mbedtls/pull/4953/commits/209b041cbf7d0b06a22c926397a0ab43ba91a847) . Please cherry-pic it , or base on it

Reply:
I just remember that there was no this function in you last patch, have you updated your patch recently?  If you didn't need this function really, please remove it from your patch, and I will add it here when it's needed.

Reply:
mbedtls_ssl_tls13_fetch_handshake_msg is rename it to mbedtls_ssl_tls1_3_fetch_handshake_msg and add it into #4977